src/ebfloeseg/load.py

```diff
@@ -37,6 +37,17 @@ def get_width_height(bbox: str, scale: float):
     return width, height
 
 
+def logger_config(debug: bool, verbose: bool, quiet: bool):
+    if debug:
+        logging.basicConfig(level=logging.DEBUG)
+    elif verbose:
+        logging.basicConfig(level=logging.INFO)
+    elif quiet:
+        logging.basicConfig(level=logging.ERROR)
+    else:
+        logging.basicConfig(level=logging.WARNING)
```

Comment:
In a separate PR I pointed out some ways to simplify this a bit by using `logging.basicConfig` once after defining a logging `level`.

Reply:
I've merged that change now